### PR TITLE
Schedule pass after DOM mutation done by render delaying services.

### LINKED ISSUE #4075: Render delaying extensions should call resources.schedulePass() after DOM mutation
As pointed out by @cramforce and @dvoytenko here:
https://github.com/ampproject/amphtml/pull/3983/files/e25e1b372db0963379f08b9f84c37fc73d2cf8f1..91bfc574cd3ef180ebad3895f2a4e58d23f897e8#r70861329

### PR DESCRIPTION
**Pull request by lannka**
Fixes #4075 and #5025

### CODE REVIEW

File: src/style-installer.js [hunk 1]
```diff
@@ -126,6 +127,8 @@ export function makeBodyVisible(doc, opt_waitForServices) {
         doc.body.style['WebkitAnimation'] = 'none';
       }
     }
+
+    resourcesForDoc(doc)./*OK*/schedulePass(1, /* relayoutAll */ true);
```
Comment (dvoytenko) [inline src/style-installer.js]: Could you protect against when `opt_waitForServices` is empty?

Comment (lannka) [inline src/style-installer.js]: Done. Added unit tests.

**Comment by zhouyx** [comment]
lgtm on my side

**Comment by dvoytenko** [comment]
LGTM

File: src/style-installer.js [hunk 2]
```diff
@@ -128,14 +129,18 @@ export function makeBodyVisible(doc, opt_waitForServices) {
...
     }
     win[bodyVisibleSentinel] = true;
     if (opt_waitForServices) {
-      waitForServices(win).then(set, set).then(() => {
+      waitForServices(win).catch(() => []).then(services => {
+        set();
+        if (services.length > 0) {
+          resourcesForDoc(doc)./*OK*/schedulePass(1, /* relayoutAll */ true);
+        }
+      }).then(() => {
```
Comment (jridgewell) [inline src/style-installer.js]: Is another `#then` block necessary?

Comment (lannka) [inline src/style-installer.js]: not all all :-) removed.

**Pull request merged**